### DISCUSSION TITLE
Tailwind HTML -> GPUI

### DISCUSSION THREAD

**Discussion by tpisto** [zed-industries/zed#8831] [Show and tell]
If you are interested about GPUI, fork my repo and enjoy HTML hot-reloading to GPUI. You can now easily design your UI using the hot-reloading HTML and then transform the layout to your components. GPUI is not fully supporting all Tailwind classes, so with this you will be 100% sure that your design will work nicely with GPUI elements.

Repo: https://github.com/tpisto/xml2gpui

**Comment by lukajincharadze** [comment]
thanks!
